### PR TITLE
stolostron/backlog#26606 internal cert 2.7

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

File: multicluster_engine/cluster_lifecycle/create_openstack.adoc [hunk 1]
```diff
@@ -80,7 +80,44 @@ Proxy information that is provided in the credential is automatically added to t
...
+stringData:
+  ca.crt: |
+    -----BEGIN CERTIFICATE-----
+    <Base64 certificate contents here>
+    -----END CERTIFICATE-----
+    -----BEGIN CERTIFICATE-----
+    <Base64 certificate contents here>
+    -----END CERTIFICATE----
+----
+
```
Comment (swopebe) [inline multicluster_engine/cluster_lifecycle/create_openstack.adoc:106]: Formatting issue here.

Comment (swopebe) [reply multicluster_engine/cluster_lifecycle/create_openstack.adoc:106]: I changed the spacing a bit to fix; resolved.

File: multicluster_engine/cluster_lifecycle/create_openstack.adoc [hunk 2]
```diff
@@ -80,7 +80,44 @@ Proxy information that is provided in the credential is automatically added to t
...
 
-When you review your information and optionally customize it before creating the cluster, you can click the *YAML* switch *On* to view the `install-config.yaml` file content in the panel. You can edit the YAML file with your custom settings, if you have any updates.  
+When you review your information and optionally customize it before creating the cluster, you can click the *YAML* switch *On* to view the `install-config.yaml` file content in the panel. You can edit the YAML file with your custom settings, if you have any updates. 
+
+When creating a cluster that uses an internal certificate authority (CA), you need to customize the YAML file for your cluster by completing the following steps:
+
+. With the *YAML* switch on at the review step, insert a `Secret` object at the top of the list with the CA certificate bundle. *Note:* If the OpenStack environment provides services using certificates signed by multiple authorities, the bundle must include the certificates to validate all of the required endpoints. The addition for a cluster named `ocp3` resembles the following example:
```
Comment (swopebe) [inline multicluster_engine/cluster_lifecycle/create_openstack.adoc]: Can you check on the proper naming of OpenStack here?

**Review by swopebe — APPROVED** [review]
small comment, fixed formatting

**Review by abutcher — APPROVED** [review]
Changes are consistent with the kcs article https://access.redhat.com/articles/6495941 and look good to me.

File: multicluster_engine/credentials/credential_openstack.adoc [hunk 3]
```diff
@@ -27,6 +27,26 @@ Start at the navigation menu. Click *Credentials* to choose from existing creden
 
 . Add your Red Hat OpenStack Platform `clouds.yaml` file contents. The contents of the `clouds.yaml` file, including the password, provide the required information for connecting to the Red Hat OpenStack Platform server. The file contents *must* include the password, which you add to a new line immediately after the `username`.
 
+. *For configurations that use an internal certificate authority* If you are using an internal certificate authority, modify your `clouds.yaml` file to reference the final location for the certificate bundle within the Hive deployer pod. Hive mounts the certificate bundle secret to `/etc/openstack-ca` within the deployer pod. The files inside of that directory correspond to the keys in the secret that is provided when you create the cluster.
```
Comment (swopebe) [inline multicluster_engine/credentials/credential_openstack.adoc]: We should not use bold here, reserve Bold for clickable and special tags (accessibility reasons). I recommend simply indicating the usage in the first line.

Comment (swopebe) [reply multicluster_engine/credentials/credential_openstack.adoc]: same with the word must in line 28.

Comment (swopebe) [reply multicluster_engine/credentials/credential_openstack.adoc]: @cadawson couple small comments, FYI.

File: multicluster_engine/cluster_lifecycle/create_openstack.adoc (hunk 2 above)
Comment (cadawson) [reply multicluster_engine/cluster_lifecycle/create_openstack.adoc]: Good catch. Thanks.

**Pull request merged**